frontend/app/src/scenes/Home/scenes/AddCategory/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {
+  Button,
+  Col,
+  Form,
+  Input,
+  notification,
+  Row,
+  Spin,
+  Typography,
+} from 'antd';
+import mainStyles from '../../styles.css';
+import { AddCategoryStateProps } from './interfaces';
+import { RouteComponentProps, withRouter } from 'react-router';
+import axios, { AxiosResponse } from 'axios';
+import { Category } from '../../interfaces';
+import { handleApiError } from '../../../../services/util/errorHandler';
+
+const { Title, Text } = Typography;
+
+class AddCategory extends React.Component<
+  RouteComponentProps,
+  AddCategoryStateProps
+> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      isLoading: false,
+    };
+  }
+
+  onFinish = (values: any) => {
+    this.setState({ isLoading: true });
+    axios
+      .post(`${window.origin}/core/academix/admin/categories`, values)
+      .then((res: AxiosResponse<Category>) => {
+        if (res.status == 201) {
+          this.setState({ isLoading: false });
+          notification.success({
+            message: 'Success!',
+            description: 'Successfully Created a Category',
+          });
+          this.props.history.push('/dashboard/academix/categories');
+        }
```

Comment:
handle non 201 codes. (add else block :) )

Reply:
Working on it!

Reply:
Resolved with commit [5c9ed7c](https://github.com/sef-global/sef-core/pull/75/commits/5c9ed7cf928bf44cff036b11cd9abb2e3ccf51e4)

Reply:
no just throw an error. the catch block will catch it.

Reply:
Ah ha, thanks for the guidance! On it!

Reply:
Resolved with commit [8d8d2ca](https://github.com/sef-global/sef-core/pull/75/commits/8d8d2ca1b7aaebafbf573bc0b20e319a0ee04d77)